mapie/conformity_scores/conformity_scores.py

```diff
@@ -158,20 +184,148 @@ def get_conformity_scores(
 
         Parameters
         ----------
+        X: NDArray
+            Observed feature values.
+
         y: NDArray
-            Observed values.
+            Observed target values.
 
         y_pred: NDArray
-            Predicted values.
+            Predicted target values.
 
         Returns
         -------
         NDArray
             Conformity scores.
         """
-        conformity_scores = self.get_signed_conformity_scores(y, y_pred)
+        conformity_scores = self.get_signed_conformity_scores(X, y, y_pred)
         if self.consistency_check:
-            self.check_consistency(y, y_pred, conformity_scores)
+            self.check_consistency(X, y, y_pred, conformity_scores)
         if self.sym:
             conformity_scores = np.abs(conformity_scores)
         return conformity_scores
+
+    @staticmethod
+    def get_quantile(
+        values: NDArray,
+        alpha_np: NDArray,
+        axis: int,
+        method: str
+    ) -> NDArray:
+        """
+        Compute the alpha quantile of the conformity scores considering
+        the symmetrical property if so.
+
+        Parameters
+        ----------
+        values: NDArray
+            Values from which the quantile is computed, it can be the
+            conformity scores or the conformity scores aggregated with
+            the predictions.
+
+        alpha_np: NDArray
+            NDArray of floats between ``0`` and ``1``, represents the
+            uncertainty of the confidence interval.
+
+        axis: int
+            The axis from which to compute the quantile.
+
+        method: str
+            ``"higher"`` or ``"lower"`` the method to compute the quantile.
+
+        Returns
+        -------
+        NDArray
+            Lower and upper quantile of the prediction intervals.
+            These quantiles are identical if the score is not symmetrical.
```

Comment:
UPD: the computed quantile is lower or upper. This description must be updated (no symmetrical aspect).

Reply:
UPD: need a description for the returns.